ui/client/http/HttpService.ts

```diff
@@ -290,11 +291,14 @@ class HttpService {
   cancel(processId, comment?) {
     return api.post(`/processManagement/cancel/${encodeURIComponent(processId)}`, comment)
       .then(() => this.addInfo(i18next.t("notification.info.scenarioCancelled", "Scenario {{processId}} was canceled", {processId})))
-      .catch(error => this.addError(
-        i18next.t("notification.error.failedToCancel", "Failed to cancel {{processId}}", {processId}),
-        error,
-        true
-      ))
+      .catch(error => {
+          if (error?.response?.status == 400) throw error
```

Comment:
Hmm, I don't think it's good idea.. Where this error will be showed?

Reply:
it's caught and shown in DeployProcessDialog